cms/urls.py

```diff
@@ -114,6 +114,12 @@
 )
 
 
+if settings.SITE_NAME == 'studio.suclass.stanford.edu':
```

Comment:
hey @kluo .  sorry, after a bit of discussion, @stvstnfrd and I think we should just create another setting, called SHIB_ONLY_SITE (and set it to false in envs.py and add an override in aws.py).  And condition this based on that setting.